patches/net/minecraft/client/renderer/chunk/SectionRenderDispatcher.java.patch

```diff
@@ -18,6 +27,27 @@
              );
              return this.lastRebuildTask;
          }
+@@ -497,6 +_,20 @@
+             );
+         }
+ 
++        // Neo: start
```

Comment:
Is this really necessary?

Reply:
It makes it a bit easier to determine what was patched in here specifically, because there are inner classes after the patched-in logic, but I can remove it if you really don't like it.